images/skopeo-gcr-io.yaml

```diff
@@ -17,6 +17,6 @@ gcr.io:
     cadvisor/cadvisor:
       - ">= v0.44.0"
     k8s-staging-cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:
-      - ">= 1.7.0"
+      - "v1.8.0"
```

Comment:
did you want a >= and no `v` here?

Reply:
i want 1.8.0 only, my bad

Reply:
fixed

Reply:
is it like that somewhere else? you might still need the `=` operator

Reply:
yeah there are other examples: https://github.com/giantswarm/retagger/blob/7ed2f2593bc63a5c264bfddbdb7afa384c7d56e1/images/skopeo-gcr-io.yaml#L5

Reply:
perfect LGTM then